packages/web/package.json

```diff
@@ -39,12 +39,12 @@
     "@angular-devkit/build-webpack": "~0.901.0",
     "@angular-devkit/core": "~9.1.0",
     "@angular-devkit/schematics": "~9.1.0",
-    "@babel/core": "^7.8.4",
-    "@babel/preset-env": "^7.8.4",
-    "@babel/preset-typescript": "^7.8.3",
-    "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@babel/plugin-proposal-decorators": "^7.8.3",
-    "@babel/plugin-transform-regenerator": "^7.8.3",
+    "@babel/core": "7.8.7",
+    "@babel/preset-env": "7.8.7",
+    "@babel/preset-typescript": "7.8.3",
```

Comment:
Is this supposed to not be `7.8.7` like the other `@babel` deps?

Reply:
No, it stops at 7.8.3, same with react. It looks like they only published packages that had changed -- including the preset-env bug.